runconfig/hostconfig.go

```diff
@@ -100,8 +100,10 @@ type RestartPolicy struct {
 }
 
 type LogConfig struct {
-	Type   string
-	Config map[string]string
+	Type         string
+	Capacity     string
+	Config       map[string]string
```

Comment:
I'm guessing the reason for `Config` is so that you can put arbitrary settings in it.  So I don't think we should add more fields to LogConfig